### PR TITLE
Added support for manually creating a phishing incident

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/41428

## Description
Added support for manually triggering phishing incidents:
- Updated `Phishing - Generic v3` playbook to handle manual incidents.
- Updated the New Incident layout form for phishing incidents.
- Moved ReporterAddress related task to the beginning of the playbook to allow some of the changes in this PR.
- Changed the Entity Enrichment playbook to use `incident.reporteremailaddress` which will allow us to get rid of the ReporterAddress key in the future (since we have an incident field for it now)

## Screenshots
New incident form

image

image

Phishing_-_Generic_v3 - Workplan

## Minimum version of Cortex XSOAR
6.1.0

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 1]
```diff
@@ -931,7 +907,7 @@
...
-                        "description": "**PCL** - Phishing Confidence Level (4-8: The message content is likely to be phishing.)\n**BCL** - Bulk complaint level (4-7: The message is from a bulk sender that generates a mixed number of complaints. 8-9: The message is from a bulk sender that generates a high number of complaints.)\n**SCL** - Spam confidence level (5-6: Spam filtering marked the message as Spam. 9: Spam filtering marked the message as High confidence spam)\n`Note`: The default value is 0, but it can be changed when the \"Process Microsoft's Email Headers\" playbook runs.",
+                        "description": "**PCL** - Phishing Confidence Level (4-8: The message content is likely to be phishing.)\\n**BCL** - Bulk complaint level (4-7: The message is from a bulk sender that generates a mixed number of complaints. 8-9: The message is from a bulk sender that generates a high number of complaints.)\\n**SCL** - Spam confidence level (5-6: Spam filtering marked the message as Spam. 9: Spam filtering marked the message as High confidence spam)\\n`Note`: The default value is 0, but it can be changed when the \\\"Process Microsoft's Email Headers\\\" playbook runs.",
```
Comment (tomer-pan) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:910]: This is a bug - the format probably escaping the escape characters - this line should stay as it was.

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 2]
```diff
@@ -1188,7 +1049,7 @@
         ]
     },
     "group": "incident",
-    "id": "Phishing Incident v3",
+    "id": "Phishing Incident_copy",
```
Comment (tomer-pan) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1052]: id should not be changed

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 3]
```diff
@@ -1489,7 +1350,7 @@
             }
         ]
     },
-    "system": true,
+    "system": false,
```
Comment (tomer-pan) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1353]: This should be true I guess

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 2 above)
Comment (idovandijk) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1052]: [SDK bug](https://github.com/demisto/etc/issues/41253), I'll fix manually

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 1 above)
Comment (idovandijk) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:910]: @dorschw is this normal?

**Comment by idovandijk** [comment]
Update: the files belong to an old version of the phishing playbook. Working on updating the newer version and will fix the PR

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 1 above)
Comment (idovandijk) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json]: Weird, my IDE shows it the way it was before

**Comment by idovandijk** [comment]
@yaron-libman can someone please review
* https://github.com/demisto/content/blob/b2b3aa61edec6a3f11fc47581653ff56a9439527/Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json
lines 992, 1015
* RN

**Review by tomer-pan — APPROVED** [review]

**Comment by yaron-libman** [comment]
I graced you with a review by me...it probably won't happen again.

**Comment by idovandijk** [comment]
@yaron-libman it seems to be bugged, do we need another label or something?

**Comment by yaron-libman** [comment]
I'm not aware of any other requirements. What is it getting stuck on in the process?

**Comment by idovandijk** [comment]
> I'm not aware of any other requirements. What is it getting stuck on in the process?

@ShahafBenYakir can you please assist with the "pull-check" step being stuck?

**Comment by ShahafBenYakir** [comment]
done

**Pull request merged**